labgrid/driver/usbloader.py

```diff
@@ -55,6 +55,7 @@ class IMXUSBDriver(Driver, BootstrapProtocol):
     }
 
     image = attr.ib(default=None)
+    verify = attr.ib(default=True)
```

Comment:
```suggestion
    verify = attr.ib(default=True, validator=attr.validators.instance_of(bool))
```
We should validate the verify argument.